packages/abstract-api/src/schema.rs

```diff
@@ -1,14 +1,19 @@
 use crate::{ApiContract, ApiError};
 use abstract_core::api::{ApiExecuteMsg, ApiQueryMsg};
-use abstract_sdk::{
-    base::{ExecuteEndpoint, InstantiateEndpoint, QueryEndpoint},
-    core::api::{ApiConfigResponse, AuthorizedAddressesResponse},
-};
-use cosmwasm_schema::{export_schema_with_title, schema_for, write_api, QueryResponses};
-use cosmwasm_std::Empty;
+use cosmwasm_schema::QueryResponses;
 use schemars::JsonSchema;
 use serde::Serialize;
-use std::path::Path;
+
+#[cfg(feature = "schema")]
```

Comment:
Instead of feature flagging these imports and stuff, can you just feature flag the `schema.rs` in `lib.rs`? Cleaner and easier to maintain.